core/src/Measurement.hpp

```diff
@@ -1,96 +1,126 @@
 /**
- * Copyright (C) 2024 Stylianos Piperakis, Ownage Dynamics L.P.
- * Serow is free software: you can redistribute it and/or modify it under the terms of the GNU
- * General Public License as published by the Free Software Foundation, version 3.
- *
- * Serow is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
- * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License along with Serow. If not,
- * see <https://www.gnu.org/licenses/>.
- **/
+ * @file Measurements.hpp
+ * @brief Defines various measurement structs used in the Serow library
+ * @author Stylianos Piperakis
+ */
+
 #pragma once
 
 #ifdef __linux__
 #include <eigen3/Eigen/Dense>
 #else
 #include <Eigen/Dense>
 #endif
+
 #include <map>
 #include <optional>
 #include <string>
 
 namespace serow {
 
+/**
+ * @struct JointMeasurement
+ * @brief Represents a joint measurement consisting of timestamp, position, and optional velocity
+ */
 struct JointMeasurement {
-    double timestamp{};
-    double position{};
-    std::optional<double> velocity{};
+    double timestamp{};                     ///< Timestamp of the measurement
+    double position{};                      ///< Joint position measurement
```

Comment:
Maybe specify angles or rads for position

Reply:
done